tests/tape/test_qnode.py

```diff
@@ -956,3 +956,31 @@ def circuit(a):
         res = circuit(0.8, shots=2)
         assert len(res) == 2
         assert dev.shots == 3
+
+
+def test_finitediff_float32(tol):

```

Comment:
I was unsure of where to put this test.  It's not actually an interface specific thing, just a float32 thing.

Reply:
Also, I don't know why `StronglyEntanglingLayers` has a problem but simpler circuits don't.

Reply:
> I was unsure of where to put this test. It's not actually an interface specific thing, just a float32 thing.

It looks fine here :) Alternatively, it could also be placed in `test_jacobian_tape` (does a file like that exist?).

> Also, I don't know why StronglyEntanglingLayers has a problem but simpler circuits don't.

Weird 🤔 

Thanks @albi3ro! So just to confirm, this test fails on master?

Reply:
test fails on master.  

I tried writing it in terms of tapes, but then you have to expand `StronglyEntanglingLayers` and update parameters and set them trainable, so I just went the simpler route of testing at QNode level.